zebedee-cms/src/test/java/com/github/onsdigital/zebedee/filters/AuthenticationFilterTest.java

```diff
@@ -1,260 +1,239 @@
+    private static final String SOME_VALIDATION_ERROR = "some error saying invalid session";
+
+    /**
+     * Class under test
+     */
+    private AuthenticationFilter authenticationFilter;
 
     @Mock
-    private Sessions sessions;
+    private HttpServletRequest request;
 
     @Mock
-    private Session session;
+    private HttpServletResponse response;
 
     @Mock
-    private Serialiser Serialiser = new Serialiser();
+    private Sessions sessions;
 
     @Mock
-    private HttpServletRequest mockRequest;
+    private ServletOutputStream outputStream;
 
-    @Before
-    public void setUp() throws Exception {
-        MockitoAnnotations.initMocks(this);
+    @Captor
+    private ArgumentCaptor<byte[]> responseBodyCaptor;
 
-        jwtHandler = new JWTHandlerImpl();
-        response   = new MockHttpServletResponse();
-        request    = new MockHttpServletRequest();
-        rsaKeyMap  = new HashMap<String, String>();
+    @Captor
+    private ArgumentCaptor<Integer> responseBodyOffset;
 
-        session = new Session();
-        session.setId(TEST_SESSION_ID);
-        session.setEmail(TEST_USER_EMAIL);
+    @Captor
+    private ArgumentCaptor<Integer> responseBodyLength;
 
-        request.setPathInfo(DUMMY_PATH);
+    @Before
+    public void setUp() throws Exception {
+        MockitoAnnotations.initMocks(this);
 
-        authFilterOldModel = new AuthenticationFilter(false, this.sessions);
-        authFilterNewModel = new AuthenticationFilter(true, jwtSessionsServiceImpl);
+        authenticationFilter = new AuthenticationFilter(sessions);
 
-        rsaKeyMap.put(RSA_KEY_ID_1, RSA_SIGNING_KEY_1);
-        jwtSessionsServiceImpl = new JWTSessionsServiceImpl(jwtHandler, rsaKeyMap);
+        doNothing().when(response).setContentType(MediaType.JSON_UTF_8.toString());
+        doNothing().when(response).setStatus(anyInt());
+        when(response.getOutputStream()).thenReturn(outputStream);
+        doNothing().when(sessions).resetThread();
     }
 
     @Test
-    public void authorisationFilterUsingLegacySessionsModelTrueTest() throws IOException {
-        when(sessions.get(request)).thenReturn(session);
-        
-        result = authFilterOldModel.filter(request, response);
-        assertEquals(result, true);
-    }
+    public void filter_ShouldReturnTrue_WhenTokenNotProvidedForOptionsRequest() throws Exception {
```

Comment:
Unsure if we have specific conventions around these test names but I would expect them to follow Java standards and be camelCase?